guides/source/action_mailbox_basics.md

```diff
@@ -277,13 +294,59 @@ class ApplicationMailbox < ActionMailbox::Base
 end
 ```
 
-Then set up a mailbox:
+The regular expression matches the incoming email's `to` field. For example, the above will match any email sent to `save@` to a 'forwards' mailbox, which we will need to create.
```

Comment:
Nice :+1:

I just remembered there's a few other ways you can add "routing" that aren't regex based, maybe we could point people to the API to learn more about them? https://api.rubyonrails.org/classes/ActionMailbox/Base.html

Reply:
Totally. Good call!